README.md

```diff
@@ -1 +1,33 @@
-# gallery
\ No newline at end of file
+# gallery
+
+Gallery provides Loom's UI for interacting with the entity data model, as well as downloading and visualizing datasets.
+
+**Setup**
+
+1. Install [cassandra](https://docs.datastax.com/en/cassandra/2.0/cassandra/install/installDeb_t.html)
```

Comment:
That link is to an old version of Cassandra on Debian. I would assume (and explicitly state) that instructions for mac so below `1.` I would include `brew install cassandra` and reference the Cassandra version we are using (3.9).